### DISCUSSION TITLE
RGB Colour picker needed

### DISCUSSION THREAD

**Discussion by techguyone** [rocksdanister/lively#940] [Ideas]
Having to manually adjust by eye is not helpful when matching themes with other components a simple RGB colour picker would help

R xxx G xxx B xxx

That kind of thing.

**Comment by rocksdanister** [comment]
Installer version of Lively supports this already:
https://github.com/rocksdanister/lively/wiki/Differences-Between-Distributions

![Screenshot 2021-12-19 014141](https://user-images.githubusercontent.com/17554161/146654415-3375b064-0f13-471a-8126-03b9e70dcfb3.png)

**Reply by techguyone** [reply]
That's great, I however, was only made aware of the app via Microsoft Store, will this one too be updated or is a cut down version?

**Reply by rocksdanister** [reply]
Yes store version is limited, you can read some of the limitations here:
https://github.com/rocksdanister/lively/wiki/Differences-Between-Distributions
